src/loader/bgw_launcher.c

```diff
@@ -90,11 +115,13 @@ bgw_on_postmaster_death(void)
 }
 
 static void
-report_bgw_limit_exceeded(void)
+report_bgw_limit_exceeded(DbHashEntry *entry)
 {
-	ereport(LOG, (errcode(ERRCODE_CONFIGURATION_LIMIT_EXCEEDED),
-				  errmsg("TimescaleDB background worker limit of %d exceeded", guc_max_background_workers),
-				  errhint("Consider increasing timescaledb.max_background_workers.")));
+	if (entry->state_advance_failures == 0)
```

Comment:
So maybe instead of `if failures == 0`, how about do something like `if failures % 10 == 0`? Because otherwise, this var doesn't seem useful because the first time (in the entire lifetime of the entry, which can be a long time if it's a user's database), we'll print an error, and then never again?